docs/source/common/trainer.rst

```diff
@@ -669,43 +669,38 @@ fast_dev_run
 
 |
 
-Runs n if set to ``n`` (int) else 1 if set to ``True`` batch(es) of train, val and test
-to find any bugs (ie: a sort of unit test).
```

Comment:
The unit test comment is very misleading considering it doesn't run any special assertions